### PR TITLE
Small updates in fuzz tests

### PR DESCRIPTION
**Pull request by lukaszstolarczuk**
This is partially in response to review comments left here: https://github.com/oneapi-src/unified-runtime/pull/795/files

### CODE REVIEW

File: test/fuzz/README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-# Corpra for fuzz tests with fixed API calls scenarios
-These corpra contain UR API calls in a predefined order described below.
+# Corpora for fuzz tests with fixed API calls scenarios
```
Comment (pbalcer) [inline test/fuzz/README.md:1]: Do people actually use the word 'corpora'? :D I get that this is the right Latin-derived plural, but it just sounds odd to me.

**Review by pbalcer — APPROVED** [review]

File: test/fuzz/README.md (hunk 1 above)
Comment (lukaszstolarczuk) [reply test/fuzz/README.md:1]: Hehe, I have no idea. It's the first time I see this word😄
Anyway, I don't think `Corpra` was a proper word - do you have a suggestion in mind?


// I found `corpora` used once, here: https://github.com/google/fuzzing/blob/master/tutorial/libFuzzerTutorial.md#other-fuzzing-engines

Comment (PatKamin) [reply test/fuzz/README.md:1]: And also in the libFuzzer docs here (also once): https://llvm.org/docs/LibFuzzer.html - `corpora` is correct.

File: test/fuzz/CMakeLists.txt [hunk 2]
```diff
@@ -14,31 +14,29 @@ function(add_fuzz_test name label)
...
-
-add_fuzz_test(kernel-launch fuzz
-    ${CORPUS_DIR}/kernel-launch -verbosity=1)
+file(GLOB files "${CORPUS_DIR}/*")
 
-add_fuzz_test(pool-alloc fuzz
-    ${CORPUS_DIR}/pool-alloc -verbosity=1)
+foreach(file ${files})
+    get_filename_component(filename ${file} NAME)
+    add_fuzz_test(${filename} fuzz-short ${file})
```
[the remark this answers is not in the document]
Comment (lukaszstolarczuk) [reply test/fuzz/CMakeLists.txt:41]: omg, I forgot it, good catch, thx!

**Review by PatKamin — APPROVED** [review]

**Pull request merged**